### PR TITLE
[core] Add Swimming DEM to Travis

### PR DESCRIPTION
**Pull request by maceligueta**
The idea is to add the Swimming DEM to Travis (or actions), but it will require activating PFEMFluid, as the Swimming DEM also couples DEM with PFEM.
I will close this PR if it takes too much compilation time.

Added later from comments below and further thoughts: 

- Swimming DEM uses Python modules which are non-standard
- PFEM Fluid uses Triangle/Tetgen

### CODE REVIEW

**Comment by philbucher** [comment]
I think sth went wrong, the title does not match the implementation, can you please check @maceligueta ?

File: scripts/build/travis/configure_travis_bionic.sh [hunk 1]
```diff
@@ -31,6 +31,9 @@ add_app ${KRATOS_APP_DIR}/DEMApplication;
 add_app ${KRATOS_APP_DIR}/CSharpWrapperApplication;
 add_app ${KRATOS_APP_DIR}/MetisApplication;
 add_app ${KRATOS_APP_DIR}/TrilinosApplication;
+add_app ${KRATOS_APP_DIR}/PfemApplication;
+add_app ${KRATOS_APP_DIR}/PfemFluidDynamicsApplication;
+add_app ${KRATOS_APP_DIR}/TrilinosApplication;
```
Comment (roigcarlo) [inline scripts/build/travis/configure_travis_bionic.sh]:
```suggestion
add_app ${KRATOS_APP_DIR}/SwimmingDEMApplication;
```

**Comment by maceligueta** [comment]
Oops, you are right @philbucher , thanks @roigcarlo too.

File: scripts/build/travis/configure_travis_bionic.sh [hunk 2]
```diff
@@ -31,6 +31,8 @@ add_app ${KRATOS_APP_DIR}/DEMApplication;
 add_app ${KRATOS_APP_DIR}/CSharpWrapperApplication;
 add_app ${KRATOS_APP_DIR}/MetisApplication;
 add_app ${KRATOS_APP_DIR}/TrilinosApplication;
+add_app ${KRATOS_APP_DIR}/PfemFluidDynamicsApplication;
```
Comment (philbucher) [inline scripts/build/travis/configure_travis_bionic.sh:34]: this one is supposed to be there?

Comment (maceligueta) [reply scripts/build/travis/configure_travis_bionic.sh:34]: Yes, the Swimming DEM app solves the coupling of DEM with Fluid, but also with PfemFluid

**Comment by maceligueta** [comment]
Running tests for PfemFluidDynamicsApplication
2983/home/travis/build/KratosMultiphysics/Kratos/bin/Custom/applications//PfemFluidDynamicsApplication/tests/test_PfemFluidDynamicsApplication.py
2984.
2985[Error]: Tests for PfemFluidDynamicsApplication took to long. Process Killed.

**Comment by maceligueta** [comment]
Hi @AFranci , @ipouplana , @jgonzalezusua , I don't know who added the PfemFluid tests. Apparently they take too long. Can any of you take a look and shorten them a bit? This is the error on Travis:
```
Running tests for PfemFluidDynamicsApplication
2983/home/travis/build/KratosMultiphysics/Kratos/bin/Custom/applications//PfemFluidDynamicsApplication/tests/test_PfemFluidDynamicsApplication.py
2984.
2985[Error]: Tests for PfemFluidDynamicsApplication took to long. Process Killed.
```

**Comment by maceligueta** [comment]
I think this PR is a profitable investment for the Swimming DEM app and the PFEM app.

**Comment by philbucher** [comment]
Also you need some skiptests in the swimmingdemapp for non-standard python modules

**Comment by maceligueta** [comment]
@philbucher . We will do. Currently we are waiting for @AFranci  to shorten the tests of the PfemFluid. He told me he would take care of that.

**Comment by roigcarlo** [comment]
I will close this as we have a new CI and eventually we will remove Travis. Still Swimming can be added to the new CI's

**Pull request closed without merging**